pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -112,19 +112,29 @@ func (s *Server) initCertController(args *PilotArgs) error {
 //
 // TODO: If the discovery address in mesh.yaml is set to port 15012 (XDS-with-DNS-certs) and the name
 // matches the k8s namespace, failure to start DNS server is a fatal error.
-func (s *Server) initDNSCerts(hostname, namespace string) error {
+func (s *Server) initDNSCerts(hostname, customHost, namespace string) error {
```

Comment:
Please use a list for customHost(s). Why restrict it to 1?

Reply:
lists actually worked too, to my surprise -- I discovered upon testing of 2 customhosts separated by ,  I will make that clear in additional PR.